examples/miral-shell/shell_main.cpp

```diff
@@ -139,6 +149,10 @@ int main(int argc, char const* argv[])
             WaylandExtensions{},
             X11Support{},
             ConfigureDecorations{},
+            pre_init(ConfigurationOption{[&](bool option)
+                    { focus_stealing_prevention = to_focus_stealing(option); },
+                    "focus-stealing-prevention", "allow or prevent focus stealing",
+                    default_focus_stealing_prevention}),
```

Comment:
Can this not require the `1`? Just be a command-line switch, rather than a valued option?